open-company/open-company-web
# Drift chat fixes for dynamic pages

Remove some warnings from the console. It wasn't a real issue since drift was already working.

To test this change the DRIFT_KEY in drift.js with the one we have in infrastructure for production.

To test:
- go to /login
- dismiss the login modal
- scroll to the footer
- click the Contact link
- [x] do you see the Drift chat popping up? Good
- go to /login
- dismiss the login modal
- click on About in the headline (not in the footer)
- click the Say Hello link
- [x] do you see the Drift chat popping up? Good
- go to /login
- dismiss the login modal
- click on Pricing in the headline (not in the footer)
- click the Contact us and Get in touch with us buttons
- [x] do you see the Drift chat popping up? Good
